src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -785,6 +785,17 @@ def __init__(self, config, *inputs, **kwargs):
     def get_output_embeddings(self):
         return self.albert.embeddings
 
+    def resize_token_embeddings(self, new_num_tokens):
+        super().resize_token_embeddings(new_num_tokens=new_num_tokens)
+
+        if new_num_tokens is not None:
+            self.predictions.bias = self.add_weight(
+                shape=(new_num_tokens,), initializer="zeros", trainable=True, name="bias"
+            )
+            self.predictions.decoder_bias = self.add_weight(
+                shape=(new_num_tokens,), initializer="zeros", trainable=True, name="bias"
+            )
```

Comment:
Are you sure that this does not erase the current value of the bias, instead replacing it by all zeros?

Reply:
I does erase yes. I thought that the resizing were happening only with a fresh new model just before the training to adapt the model the vocab size it has to be trained on. Something like:
```
from transformers import TFBertMaskedLM, BertConfig

config = BertConfig()
model = TFBertMaskedLM(config)
model.resize_token_embeddings(new_size)
```

But I can make not erasing the current bias values, it is not a problem :)